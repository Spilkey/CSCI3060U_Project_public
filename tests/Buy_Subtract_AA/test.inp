login
user01
buy
airpods meetup
user02
10
yes
logout
exit