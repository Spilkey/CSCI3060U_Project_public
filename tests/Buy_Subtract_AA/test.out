04_airpods_meetup____________user02________010_100.00
00_user01__________AA_000000.00